copier/main.py

```diff
@@ -154,6 +154,11 @@ class Worker:
 
         conflict:
             One of "rej" (default), "inline" (still experimental).
+
+        update_context:
+            When `True`, ignores answers file directory when updating.
+
+            See [update_context][].
```

Comment:
Hello! I've reviewed this, but I cannot understand the patch. Quoting from the discussion:

> to be able to store the answer files in a sub-directory

So this is a rather small scope AFAICS.

What's this flag for, `update_context`? What relation does it have with the fact that an answers file is or not in a directory?

I'm a bit confused with this patch... 😅

Reply:
Hi Yajo! Sorry for the late response! When I was developing this, I saw that when updating an already created template, the paths for the answer file did not resolve to the correct directory compared to generating a new project. Therefore I included the update_context to account for this difference. Maybe I did not understand the difference and made the wrong choice, but let me know how I can help / improve!

Reply:
I appreciate the effort you put into creating a patch for this issue. However, upon review, it seems that your patch does not address the issue at hand. It might be more effective to start fresh and consider writing a test first before implementing a solution. This can help ensure that the solution effectively addresses the issue.

Reply:
Hi @yajo thanks for your feedback! I will close this pull request and re-think how I could potentially solve this.